config/initializers/00_cf_services.rb

```diff
@@ -0,0 +1,7 @@
+if ENV['VCAP_SERVICES']
+  VCAP_SERVICES = JSON.parse(ENV['VCAP_SERVICES'])
```

Comment:
Is it worth `.fetch`ing these so we it raises an exception here if the variable isn't set?  I sometimes find it hard to track down bugs caused by unset variables due to the fact that `[]` assigns `null` rather than failing. 

In other words `ENV.fetch('VCAP_SERVICES')`.

Reply:
These services are only set on CloudFoundry, when we run the server on development machine we do not have them, so we do not want to raise an exception in that case